### ISSUE TITLE
Transformers `pipeline` API Support

### ISSUE THREAD

**Issue by ndebuhr** [huggingface/optimum-tpu#84]
The text generation example in the repository involves some lower-level work with logits, tensors, etc.

If optimum-tpu was compatible with the transformers Pipeline API, that would significantly ease adoption. I'm working on a specific use case (on HF Spaces with TPUs) where this would be very helpful.

**Comment by tengomucho** [comment]
This project is getting archived, closing this issue.
